src/context/hooks/useAuth.ts

```
@@ -10,6 +10,29 @@ export function useAuth() {
+    if (token && userString) {
+      try {
+        const user: AuthenticatedUser = JSON.parse(userString)
+        if (user.username && Array.isArray(user.authorities)) {
+          setAuthenticatedUser(user)
+          setAuthenticated(true)
+          api.defaults.headers.common['Authorization'] = `Bearer ${token}`
+        } else {
+          localStorage.removeItem('token')
+          localStorage.removeItem('user')
+        }
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)